src/OpenTelemetry.Instrumentation.Process/README.md

```diff
@@ -0,0 +1,39 @@
+# Process Instrumentation for OpenTelemetry .NET
+
+[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.Process.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.Process.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)
+
+This is an [Instrumentation
+Library](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library),
+which instruments [.NET Process](https://docs.microsoft.com/dotnet) and
+collect telemetry about Process behavior.
```

Comment:
```suggestion
which instruments [.NET](https://docs.microsoft.com/dotnet) process and
collect telemetry about process behavior.
```

Nit: No need to use upper case for process.

Reply:
updated; thanks!